cypress/plugins/index.js

```diff
@@ -9,6 +9,12 @@
 // ***********************************************************
 
 module.exports = (on, config) => {
+	on('task', {
+		log(message) {
+			console.log(message);
+			return null;
+		},
+	});
```

Comment:
Is this needed?

Reply:
It's not explicitly needed here, but it's a useful script to have in place as without it you can't debug the output from the GH Actions. 

I'll remove it from this PR and move it into a separate one.